v1 (#36)
cosmos/new-cosmos.network: content/articles/what-is-a-validators-commission.md

  - blockchain
  - atom
---

In the process of staking, rewards are generated proportional to the amount of ATOM staked. Of these rewards, a certain percentage goes to the validator to which the ATOM are delegated. This percentage is called the commission, and is set by validator operator themselves. 

> For example, if an ATOM holder delegates all their ATOM to a single validator with a commission of 10%, then 10% of this holder's rewards will go to that validator. Commission is therefore an important parameter for ATOM holders to take into account in deciding which validator(s) to delegate to.  Note that some validators may apply higher commission because they operate a more complex setup, meaning they incur higher operating costs (see Risks). It is the responsibility of the delegator to assess the commission of validators they delegate to with regards to the service they offer.

Reviewer: ```suggestion
For example, if an ATOM holder delegates all their ATOM to a single validator with a commission of 10%, then 10% of this holder's rewards go to that validator. Commission is therefore an important parameter for ATOM holders to take into account in deciding which validator or validators to delegate to.  Note that some validators apply a higher commission because they operate a more complex setup and incur higher operating costs (see Risks). It is the responsibility of the delegator to assess the commission of validators they delegate to with regard to the service they offer.
```
